src/main/java/Menu.java

```diff
@@ -1,40 +1,60 @@
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
+import java.util.function.Supplier;
 
 public class Menu {
 
     private UserInterface gameUI;
-
-    public enum Options {
-        TWO_HUMANS, HUMAN_AND_ROBOT, TWO_ROBOTS, HUMAN_AND_ALIEN
-    }
+    private HashMap<String, Supplier<List<Player>>> menuOptions;
+    private ArrayList<String> printedMenuOptions;
```

Comment:
You would want to use `Map` (`java.util.Map`) and `List` (`java.util.List`) for the declaration of these. It is a good practice to use the interfaces instead of the concrete classes when you define your variables.